### PR TITLE
Clean code 5 - Formatting

### PR DESCRIPTION
**Pull request by wooooooood**

### CODE REVIEW

File: clean_code/Chapter 5/Formatting.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+- 아래와 같이 `while` / `for`의 body가 빈 경우가 있다. 가능한 없애지만, 없앨 수 없는 경우에는 braces로 감싸고 적절한 indentation을 준다.
+
+```java
+while (dis.read(buf, 0, readBufferSize) != -1)
+  ;
+```
+
+## Team Rules
+
+- A team of developers should agree upon a single formatting style, and then every member of that team hould use that style.
```
Comment (AnneMayor) [inline clean_code/Chapter 5/Formatting.md]: mis spell. You should add s into `hould` 😊

**Review by AnneMayor — CHANGES_REQUESTED** [review]

File: clean_code/Chapter 5/Formatting.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+# 5. Formatting
+
+- You should take care that your code is nicely formatted. You should choose a set of
+  simple rules that govern the format of your code, and then you should consistently apply
+  those rules.
+
+## The Purpose of Formatting
+
+- `communication`: communication is the professional developer’s first order of business.
+- Your style and discipline survives, even though your code does not
```
Comment (AnneMayor) [inline clean_code/Chapter 5/Formatting.md]: survives -> `survive`

**Review by AnneMayor — CHANGES_REQUESTED** [review]

**Review by deveeni — APPROVED** [review]
Great!

File: clean_code/Chapter 5/Formatting.md (hunk 2 above)
Comment (wooooooood) [reply clean_code/Chapter 5/Formatting.md]: Copy&Pasted from book actually.. but you're right!

**Review by AnneMayor — APPROVED** [review]
LGTM 👍 Great job

**Pull request merged**